test/e2e/runner/rpc.go

```diff
@@ -63,7 +105,12 @@ func waitForHeight(testnet *e2e.Testnet, height int64) (*types.Block, *types.Blo
 			if maxResult == nil {
 				return nil, nil, errors.New("chain stalled at unknown height")
 			}
-			return nil, nil, fmt.Errorf("chain stalled at height %v", maxResult.Block.Height)
+
+			return nil, nil, fmt.Errorf("chain stalled at height %v [%d of %d nodes]",
```

Comment:
It may be even more helpful to list which nodes didn't make the height

Reply:
I did this during debugging, but it's a map, rather than a slice so it looks gross, and I think in the cases where this runs for real, we'll be able to deduce from logs. 

Deployed correctly, this is more about the waiting so the next thing in a test can happen, rather than as an assertion.